docs/core-blockchain-concepts/proof-of-stake/stake-validation-height.md

```diff
@@ -1,3 +1,29 @@
 # <img class="dcr-icon" src="/img/dcr-icons/TicketLive.svg" /> Stake Validation Height
 
 ---
+
+Stake Validation Height (SVH) is one of the [Blockchain Parameters](../blockchain-parameters.md).
```

Comment:
This all seems technically correct and is readable. However, as a noob, I'm still confused as to what exactly SVH is. It seems like it's just the block before the current height? I.e. the block before the latest block found by PoW miners? It's not clear were the 4096 comes into play....Does this mean that tickets must wait for the SEH, then wait another 4096 blocks to vote? These questions sound dumb writing them, but hopefully illustrate confusion others with have.

Reply:
Reworded to explain a bit more clearly:

> SVH is the height at which votes are required to add a new block to the top of the blockchain. On mainnet, the SVH is 4,096. This means the first 4,096 blocks created by Proof-of-Work (PoW) miners were created without including votes from Proof-of-Stake (PoS) voters. All blocks mined after block 4,096 must include at least 3 votes, otherwise they will not be considered valid by consensus rules.

Reply:
A few reasons this is useful information. 

1. As you mention, simnet and regnet both start from genesis so this is relevant for those
1. testnet is occasionally reset from genesis (we are currently on testnet3).
1. Tools such as dcrdata or dcrvotingweb which look back at historic chain data need to understand this mechanism